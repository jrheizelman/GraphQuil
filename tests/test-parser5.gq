/*
 *Test node creation
 */

void main(){
	int g;
	int out;
	out = 7;

	if(out == 7)	{
		int a;
		out = 8;
	}
	
	
	
	

	testfunc();
}

int testfunc(){
		int k;
		k = 4;
		
		k = g;
		return k;
}