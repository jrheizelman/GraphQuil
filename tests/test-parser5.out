int testfunc()
{
int k;
k = 4;
k = g;
return k;
}

Void main()
{
int g;
int out;
out = 7;
if (out == 7)
{
int a;
out = 8;
}
testfunc();
}